app/views/support/responsible_bodies/show.html.erb

```diff
@@ -15,6 +15,15 @@
       <%= @responsible_body.name %>
     </h1>
 
+    <%- if @responsible_body.extra_mobile_data_requests.present? %>
+      <h2 class="govuk-heading-l govuk-!-margin-top-9">Extra Mobile Data Requests</h2>
```

Comment:
```suggestion
      <h2 class="govuk-heading-l govuk-!-margin-top-9">MNO requests</h2>
```

Reply:
Or "Mobile data requests"

Reply:
They're referred to as "Extra mobile data requests" in the support/schools interface at the moment, hence labelling them as the same here. I'm happy to change that, but we should be consistent - so I'll need to change them both.

Reply:
Done in eae32f0